docs/output.md

```diff
@@ -24,6 +24,18 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 
 ### Data model
 
+Metapep uses a relational data model that consists of tables that can describe relationships between different objects.
+
+<p align="center">
+    <img src="images/metapep_datamodel.png" alt="nf-core/metapep data model" width="90%">
```

Comment:
I'm not sure how 90% look like in a browser, but for my preview it looks a bit large. Maybe scale to 60% width?

Reply:
https://github.com/skrakau/metapep/blob/update_readme_datamodel/docs/output.md